asciidoc/courses/aura-dashboards/modules/1-getting-started/lessons/1-what-are-dashboards/lesson.adoc

```diff
@@ -1,26 +1,31 @@
-= Understanding Aura Dashboards
+= Understanding Neo4j Dashboards
 :type: lesson
 :order: 1
 
+// TODO: Consider restructuring this lesson to focus only on what dashboards are and when to use them.
+// Currently includes course overview (lines 7-21) and prerequisites (lines 28-46) that could be moved to course.adoc.
+// This would make the lesson more focused and under 5 minutes to read.
+
 == Understanding Data Visualization 
 
-Welcome to the Visualizing Graph Data using Aura Dashboards Fundamentals course on GraphAcademy!
+Welcome to the Building Dashboards in Neo4j course on GraphAcademy!
 
 Ever wondered how movie recommendation systems work, or how studios track which genres are trending? 
 
-Representing data in a visual format can help uncover patterns that might be missed otherwise. 
+Representing data in a visual format can help uncover patterns that might be missed otherwise.
 
-* That's the power of graph databases and data visualization!
+Data visualization transforms your graph into actionable insights that anyone can understand.
 
-// Throughout this course, you will learn how to get started with Neo4j Aura Dashboards. 
+// TODO: These are course objectives, not lesson objectives. Consider adding lesson-specific objectives like:
+// "In this lesson, you will learn: What Neo4j Dashboards are, When to use dashboards, How dashboards fit into your workflow"
 
 By the end of this course, you will be confident in:
```

Comment:
[personal preference & minor point]
I would use a more positive set of statement, you will be able to, you will understand, etc.